srcpkgs/python3-bibtexparser/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'python3-bibtexparser'
 pkgname=python3-bibtexparser
-version=1.1.0
-revision=5
+version=1.2.0
+revision=1
 wrksrc="bibtexparser-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-parsing python3-future"
+depends="python3-parsing"
+checkdepends="python3-future $depends"
 short_desc="Python3 package for parsing BibTeX"
 maintainer="xaltsc <xaltsc@protonmail.ch>"
 license="LGPL-3.0-only"
 homepage="https://github.com/sciunto-org/python-bibtexparser"
```

Comment:
```suggestion
homepage="https://pypi.org/project/bibtexparser/"
```
Thanks to this, update-check will be able to detect new versions even when they are only pushed to pypi and not github.

Reply:
@paper42 Ok, I pushed the changes you suggested. Marking this as resolved.